### DISCUSSION TITLE
Better support for storage boxes - new DB field and UI elements

### DISCUSSION THREAD

**Discussion by hzolt** [sysadminsmedia/homebox#250] [Ideas]
I have started to use HomeBox and it would be very nice if it could support storage boxes better. At the moment the 'Description' field is quite limited to listing all the contents of a larger box (these are small items that I don't want to create individually - is seems like overkill to me...). I usually get an error when trying to save a longer list, probably because of the DB constraints... Now I'm using the 'Notes' as an extension to the 'Description', like '<b>CONTINUES...</b>' and <b>...CONTINUES:</b>. It's working but not ideal...

**So I'm suggesting to:**
- create a new, bigger field for boxes to list the content, maybe it should be a bulleted list out-of-box, or support HMTL formatting 
- have a checkbox to mark the item as a box and trigger to display the new field 
- have a chekcbox on the 'Search' screen for boxes

**Comment by tankerkiller125** [comment]
Looks like the description field has a limit of 1000 characters. Notes should be the SQLite limit (if I'm reading the code and understanding how it all works correctly), which from my understanding is nearly 1GB in length.

My honest recommendation in this case would be to not use the description and instead only use the Notes field if you intend to use it in this way. However, it should be noted, that the software wasn't really designed for this, and frankly the database performance (especially if those fields can be queried for something like search) will start degrading significantly if this is done a lot.

**Reply by hzolt** [reply]
First of all, thanks for your quick check and response!

I have tried the 'Notes' as well, but for a longer content I always get the "Failed to save item" error ({"error":"Unknown Error"}). Maybe it is the same type as the 'Description', but should not it be?

For my use case I think I wolud have about 20 big boxes at most.

First I started to link unique Google Sheets pages to these boxes, but in this case I have to search in at least two different places, so I switched to the 'Description' + 'Notes' combo. My main goal is to have one place where I can store and search everything :).
